projects/kit/components/select/native-select/native-select-group.component.ts

```diff
@@ -0,0 +1,43 @@
+import {ChangeDetectionStrategy, Component, Input, TemplateRef} from '@angular/core';
+import {TuiDataListDirective} from '@taiga-ui/core';
+
+import {AbstractTuiNativeSelect} from './native-select';
+
+@Component({
+    selector: `select[tuiSelect][labels]`,
+    templateUrl: `./native-select-group.template.html`,
+    providers: [
+        {
+            provide: TuiDataListDirective,
+            deps: [TuiNativeSelectGroupComponent],
+            useExisting: TuiNativeSelectGroupComponent,
+        },
+        {
+            provide: TemplateRef,
+            deps: [TuiNativeSelectGroupComponent],
+            useFactory: ({datalist}: TuiNativeSelectGroupComponent) => datalist,
+        },
+        {
+            provide: AbstractTuiNativeSelect,
+            useExisting: TuiNativeSelectGroupComponent,
+        },
+    ],
+    host: {
+        '[attr.aria-invalid]': `host.invalid`,
+        '[attr.disabled]': `host.disabled || null`,
+        '[tabIndex]': `host.focusable ? 0 : -1`,
+        '[class._mobile]': `isMobile`,
```

Comment:
I don't see `isMobile` here or in an abstract class. Why doesn't it throw an error? Or am I just blind? :)

Reply:
omg seems like you can write any bullshit there, ts doesn't check it 😱 
it was old solution that I forgot to remove, fixed

Reply:
angular-eslint has `[no-host-metadata-property](http://codelyzer.com/rules/no-host-metadata-property)`
<img width="843" alt="image" src="https://user-images.githubusercontent.com/12021443/198686670-835141bb-6b9f-467b-a32c-cbd3e0146b27.png">

Reply:
We like host metadata. I was pretty sure it is typechecked. Gotta investigate 🙂